### PR TITLE
Fix panel device header

### PR DESCRIPTION
**Pull request by InsaneSplash**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by InsaneSplash** [comment]
BEFORE:
image

AFTER:
image

File: html/index.php [hunk 1]
```diff
@@ -129,7 +129,7 @@
   <link href="css/leaflet.awesome-markers.css" rel="stylesheet" type="text/css" />
   <link href="css/select2.min.css" rel="stylesheet" type="text/css" />
   <link href="<?php echo($config['stylesheet']);  ?>?ver=291727421" rel="stylesheet" type="text/css" />
-  <link href="css/<?php echo $config['site_style']; ?>.css?ver=632417642" rel="stylesheet" type="text/css" />
+  <link href="css/<?php echo $config['site_style']; ?>.css?ver=632417692" rel="stylesheet" type="text/css" />
```
Comment (laf) [inline html/index.php]: This isn't needed when no css files are changed or did you miss committing that change?

Comment (InsaneSplash) [reply html/index.php]: :) Did then reverted them back... so forgot to revert this back too :)

**Comment by laf** [comment]
I've tested this with mono and dark themes and don't get the change here. Can you explain in what scenario this is supposed to work?

**Comment by InsaneSplash** [comment]
Very strange... Its basically the sub-menu bar within the device information pages that seems to have been missed with regards to the theme. The highlight is correct, but the background has not been set.... Ill dbl check my work :)

**Comment by laf** [comment]
Hmm, might have me being blind but seems ok now.

**Pull request merged**